prototype/src/main/webapp/script.js

```diff
@@ -0,0 +1,39 @@
+PROPOSED_TIMES_NO = 3; // Initially we hardcoded 3 different times suggestions
```

Comment:
Use `let`, also keep this camelCase as it is not a constant.

Reply:
Yep, done!